src/components/Application-Header/JobsAppBar.tsx

```diff
@@ -31,7 +35,7 @@ const JobsAppBar: React.FC<AppBarDataProps> = (props): JSX.Element => {
                     <FavoriteList/>
                 </Tooltip>
                 <Avatar className={classes.avatar}>
-                    {user.userInitials}
+                    {userStore.displayName.substring(0,1)}
```

Comment:
Actually... I was sure the initials are first letter of `first name` and `last name`, not the first `2 characters` of 1st name. 🤔

Reply:
I used the first letter for ease of use 😅   
You can open a new task in trello for it, although I belive the user's picture should be up there

Reply:
Merge with `Michael`, will be done 😄